vars/iqeUtils.groovy

```diff
@@ -356,6 +377,18 @@ def runIQE(String plugin, Map appOptions) {
         if (errorMsgSequential || errorMsgParallel) {
             error("${errorMsgSequential} ${errorMsgParallel}")
         }
+
+        if (screenshotsDir) {
+            lsStatus = sh(
+                script: (
+                """
+                ls -lR ${screenshotsDir}
+                """.stripIndent()
+                ),
+                returnStatus: true
+            )
+            archiveArtifacts "${screenshotsDir}/*.png"
```

Comment:
@bsquizz, @quarckster - I've tried all kinds of globbing here but nothing seems to work, yet the `ls` above shows me that the screeshot files are actually on disk. What am I missing ?

Reply:
That is a bit weird. Is screenshotsDir an absolute path or relative? Maybe try:
```
dir(screenshotsDir) {
    archiveArtifacts "*.png"
}
```

Also, are there any subdirectories in there, or just .png files?

Reply:
> Is screenshotsDir an absolute path or relative?

@bsquizz it is an absolute path. No subdirectories inside, only the PNG file. Will try your snippet.

Reply:
> The syntax might need to be:
> `archiveArtifacts artifacts: "${screenshotsDir}/*.png"`

That's odd but I can give it a try as well. Looking at the rest of the code the majority of the calls to `archiveArtifacts` don't specify the `artifacts` argument so that shouldn't be necessary: 

```
vars/iqeUtils.groovy:            archiveArtifacts "${screenshotsDir}/*.png"
vars/iqeUtils.groovy:        archiveArtifacts "iqe-${plugin}-*.log"
vars/iqeUtils.groovy:    archiveArtifacts "ibutsu.html"
vars/openShiftUtils.groovy:        archiveArtifacts(artifacts: fileName)
vars/openShiftUtils.groovy:            archiveArtifacts "oc_export_all.yaml"
vars/openShiftUtils.groovy:            archiveArtifacts "applogs/*.log"
vars/pythonUtils.groovy:    archiveArtifacts("pipenv_install_out.txt")
vars/pythonUtils.groovy:        archiveArtifacts 'htmlcov/*'
vars/clientUtils.groovy:    archiveArtifacts artifacts: '*.txt'
```

Reply:
From the docs: 

> You can only archive files that are located in your workspace

I think that may be the problem. The artifacts dir is ` /home/jenkins/agent/screenshots-137` while the workspace is `/home/jenkins/agent/workspace/image-builder-debug-screenshots/`